### ISSUE TITLE
Is it possible to use value for "recipient" of service from message payload?

### ISSUE THREAD

**Issue by bletvaska** [mqtt-tools/mqttwarn#689]
hello

this is not issue - it is a question (and i don't know where to write it, so i am writing here)

i am new to mqttwarn and i am thinking about to use it. i understand how it works, but my requirement is bit different. simply - i need to have option to send message (currently push notification with pushbullet) to recipient, which is specified in mqtt payload/message. so for example - the message should looks like this:

```json
{
   "recipient": "test@example.org",
   "message": "hello world"
}
```

and the question is, if it is possible to send the message `"hello world"` dynamically to the recipient with email `test@example.org` instead of statically defined targets in service configuration? according to the documentation and the options/functions, which i am able to change/write, it looks to me, it is not possible :-( 

thanks

mirek

**Comment by jpmens** [comment]
No, I don't think it's currently possible to override configuration dynamically,  so what you need would not easily be implemented with mqttwarn.

**Comment by bletvaska** [comment]
hmm... that's pitty. so i need to implement it by my own :-( thanks anyway

**Comment by amotl** [comment]
Hi Mirek,

thank you for writing in. Your request sounds similar like another one. However, this one is different, so I am re-opening in order to track your request appropriately.

- GH-482

With kind regards,
Andreas.
